lib/api_browser/app/views/controller.html

```diff
@@ -5,7 +5,7 @@
   <div class="row">
     <div class="col-lg-12">
       <h1 class="page-header">
-        {{ controller.name | resourceName }}
+        {{ controller.display_name | resourceName }}
```

Comment:
ah...now that I understand this better, I guess ideally what we need is something like
```
if c.display_name
  c.display_name
else
 c.name | resourceName
end
```

(However this is best expressed in the template).